ale_linters/go/gometalinter.vim

```diff
@@ -12,8 +13,16 @@ function! ale_linters#go#gometalinter#GetCommand(buffer) abort
     let l:executable = ale_linters#go#gometalinter#GetExecutable(a:buffer)
     let l:filename = expand('#' . a:buffer)
     let l:options = ale#Var(a:buffer, 'go_gometalinter_options')
+    let l:lint_dir = ale#Var(a:buffer, 'go_gometalinter_lint_dir')
+
+    if l:lint_dir
+        return ale#Escape(l:executable)
+        \   . (!empty(l:options) ? ' ' . l:options : '')
+        \   . ' ' . ale#Escape(fnamemodify(l:filename, ':h'))
+    endif
```

Comment:
Maybe an `if..else` is a bit cleaner/clearer here? I know in Go `if...else` is not done, but in vimscript it's a standard practice and personally I think that in this piece of code it's a better fit...